net.brinkervii.grapejuice.yml

```diff
@@ -0,0 +1,116 @@
+      # edit robloxapp.desktop
+      - desktop-file-edit /app/share/applications/net.brinkervii.grapejuice.robloxapp.desktop --set-icon="net.brinkervii.grapejuice.robloxapp" --set-key="Exec" --set-value="grapejuice app"
+      # edit robloxplayer.desktop
+      - desktop-file-edit /app/share/applications/net.brinkervii.grapejuice.robloxplayer.desktop --set-icon="net.brinkervii.grapejuice.robloxplayer" --set-key="Exec" --set-value="grapejuice player %u"
+      # edit robloxstudio.desktop
+      - desktop-file-edit /app/share/applications/net.brinkervii.grapejuice.robloxstudio.desktop --set-icon="net.brinkervii.grapejuice.robloxstudio" --set-key="Exec" --set-value="grapejuice studio %u" 
+      # install grapejuice icons
+      # 64x64 icons
+      - install -Dm644 src/grapejuice_common/assets/icons/hicolor/64x64/apps/grapejuice-roblox-player.png /app/share/icons/hicolor/64x64/apps/net.brinkervii.grapejuice.robloxplayer.png
+      - install -Dm644 src/grapejuice_common/assets/icons/hicolor/64x64/apps/grapejuice-roblox-studio.png /app/share/icons/hicolor/64x64/apps/net.brinkervii.grapejuice.robloxstudio.png 
+      - install -Dm644 src/grapejuice_common/assets/icons/hicolor/64x64/apps/grapejuice-roblox-player.png /app/share/icons/hicolor/64x64/apps/net.brinkervii.grapejuice.robloxapp.png
+      - install -Dm644 src/grapejuice_common/assets/icons/hicolor/64x64/apps/grapejuice.png /app/share/icons/hicolor/64x64/apps/net.brinkervii.grapejuice.png
+      # 128x128 icons     
+      - install -Dm644 src/grapejuice_common/assets/icons/hicolor/128x128/apps/grapejuice-roblox-player.png /app/share/icons/hicolor/128x128/apps/net.brinkervii.grapejuice.robloxplayer.png
+      - install -Dm644 src/grapejuice_common/assets/icons/hicolor/128x128/apps/grapejuice-roblox-studio.png /app/share/icons/hicolor/128x128/apps/net.brinkervii.grapejuice.robloxstudio.png 
+      - install -Dm644 src/grapejuice_common/assets/icons/hicolor/128x128/apps/grapejuice-roblox-player.png /app/share/icons/hicolor/128x128/apps/net.brinkervii.grapejuice.robloxapp.png
+      - install -Dm644 src/grapejuice_common/assets/icons/hicolor/128x128/apps/grapejuice.png /app/share/icons/hicolor/128x128/apps/net.brinkervii.grapejuice.png
+      # scalable icons      
+      - install -Dm644 src/grapejuice_common/assets/icons/hicolor/scalable/apps/grapejuice.svg /app/share/icons/hicolor/scalable/apps/net.brinkervii.grapejuice.svg
+      # install metainfo
+      - install -Dm644 net.brinkervii.grapejuice.metainfo.xml -t /app/share/metainfo 
+    sources:
+      - type: file
+        url: https://gitlab.com/brinkervii/wine-builds/-/raw/main/debuntu-wine-tkg-staging-fsync-git-7.1.r2.gc437a01e.tar.zst?inline=false
+        sha512: b60dd9628a592bef596049ff144ccdb7c2518afdbc70e32464f0d519122de4b2870ad8b7049f91acb876d302cde9189e899d54b615136d3bae1c0cc1bc2810da
+      - type: git
+        url: https://gitlab.com/brinkervii/grapejuice.git
+        tag: v5.1.1
```

Comment:
can we get this updated to 5.2.1 since there's a built in version checker?

Reply:
updated